src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -2246,11 +2254,20 @@ void EIRFuelFiredHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
     // calculate source side outlet conditions
     Real64 CpSrc = 0.0;
     if (this->waterSource) {
+        auto &thisSourcePlantLoop = state.dataPlnt->PlantLoop(this->sourceSidePlantLoc.loopNum);
         CpSrc = FluidProperties::GetSpecificHeatGlycol(
-            state, thisLoadPlantLoop.FluidName, thisInletNode.Temp, thisLoadPlantLoop.FluidIndex, "PLFFHPEIR::simulate()");
+            state, thisSourcePlantLoop.FluidName, this->sourceSideInletTemp, thisSourcePlantLoop.FluidIndex, "PLFFHPEIR::simulate()");
     } else if (this->airSource) {
         CpSrc = Psychrometrics::PsyCpAirFnW(state.dataEnvrn->OutHumRat);
     }
+    this->sourceSideCp = CpSrc;
+    SetupOutputVariable(state,
+                        "Heat Pump Source Side Specific Heat",
+                        OutputProcessor::Unit::J_kgK,
+                        this->sourceSideCp,
+                        OutputProcessor::SOVTimeStepType::System,
+                        OutputProcessor::SOVStoreType::Average,
+                        this->name);
```

Comment:
You putting SetupOutputVariable in doPhysics?

Reply:
What is happening why is that there lol. Good catch @rraustad !